Dockerfile.train

```diff
@@ -1,39 +1,67 @@
-# Please refer to the TRAINING documentation, "Basic Dockerfile for training"
+# This is a Dockerfile useful for training models with Coqui STT.
+# You can train "acoustic" models with audio + Tensorflow, and
+# you can train "language" models with text + KenLM.
```

Comment:
language models -> scorers

Reply:
I still think scorer is a sub-optimal name for this. The core is an n-gram language model. Where does the term "scorer" come from?

Reply:
It's an external scorer for choices during decoding. The goal is to make it extra clear that the input to our clients is /not/ a KenLM language model, but a scorer created from a KenLM language model + a trie + alphabet + metadata. Basically avoiding name overloading.

Reply:
`s/language model/scorer/`